build.gradle

```
@@ -0,0 +1,169 @@
+plugins {
+    id 'com.palantir.docker' version '0.25.0'
+    id 'java'
+    id 'application'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.21'
+    id 'com.google.protobuf' version '0.8.16'
+    id 'com.github.harbby.gradle.serviceloader' version '1.1.5'
+}
+
+ext {
+    sharedDir = file("${project.rootDir}/shared")
+    sailfishVersion = '3.2.1594'
+    genDir = 'gen/'
+    genTestGrpc = genDir + 'test/grpc'
+    genTestJava = genDir + 'test/java'
+}
+
+group = 'com.exactpro.th2'
+version = release_version
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+repositories {
+    maven {
+        name 'MavenLocal'
+        url sharedDir
+    }
+    mavenLocal()
+    mavenCentral()
+    maven {
+        name 'Sonatype_snapshots'
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+    }
+    maven {
+        name 'Sonatype_releases'
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+    }
+    maven {
+        name 'Nexus'
+        url 'https://nexus.exactpro.com/repository/oss-maven'
+    }
```

```suggestion
    maven {
        name 'Nexus'
        url 'https://nexus.exactpro.com/repository/oss-maven'
    }
```
this repository can't be used in github